flask_dance/contrib/azure.py

```diff
@@ -15,7 +15,7 @@
 def make_azure_blueprint(
         client_id=None, client_secret=None, scope=None, redirect_url=None,
         redirect_to=None, login_url=None, authorized_url=None,
-        session_class=None, backend=None):
+        session_class=None, backend=None, tenant=None):
```

Comment:
Instead of defaulting to `None`, can you have this parameter default to "common"? That way, we can take out the `tenant = tenant or "common"` line later on.

I have this construction for the `scope` argument because it's a list, which is a mutable type, and setting a mutable type as a default value for a function argument is a bad idea in Python. But strings are immutable, so it's simpler to just use it as a default value for the argument.